layouts/partials/meta/standard.html

```diff
@@ -31,14 +31,18 @@
   <link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}" title="{{ .Language.LanguageName }}" />
 {{ end }}
 
-{{ with .Params.image }}
-  <meta itemprop="image" content="{{ . | absURL }}" />
-  <meta property="og:image" content="{{ . | absURL }}" />
-  <meta name="twitter:image" content="{{ . | absURL }}" />
-  <meta name="twitter:image:src" content="{{ . | absURL }}" />
-{{ else }}
-  <meta itemprop="image" content="{{ .Site.Params.ogimage | absURL }}" />
-  <meta property="og:image" content="{{ .Site.Params.ogimage | absURL }}" />
-  <meta name="twitter:image" content="{{ .Site.Params.ogimage | absURL }}" />
-  <meta name="twitter:image:src" content="{{ .Site.Params.ogimage | absURL }}" />
-{{ end }}
+{{/*  Define empty variable image  */}}
+{{- $image := "" -}}
+
+{{/*  If image is defined in front matter, use it  */}}
+{{- if .Params.image -}}
+  {{- $image = .Params.image -}}
+{{/*  If image is not defined in front matter, use site ogimage  */}}
+{{- else if .Site.Params.ogimage -}}
+  {{- $image = .Site.Params.ogimage -}}
+{{- end -}}
+
+<meta itemprop="image" content="{{ $image | absURL }}" />
+<meta property="og:image" content="{{ $image | absURL }}" />
+<meta name="twitter:image" content="{{ $image | absURL }}" />
+<meta name="twitter:image:src" content="{{ $image | absURL }}" />
```

Comment:
Hello, @mattcen. I like your proposal. Thanks for it.

I just have a concern.

What if there is no image in the frontmatter or `ogimage`? 
Could you please ensure that the meta tags are loaded only in the case any of the 2 things are defined.
This could prevent errors from being shown.

Reply:
I *can* change this, but I'd like to point out that the behaviour resulting from my current changes is equivalent to the existing behaviour:

https://github.com/hugo-sid/hugo-blog-awesome/blob/db346b0d13bc1c6d13697e5f66be7fc7b65600ec/layouts/partials/meta/standard.html#L34-L44

The existing code already includes the meta tags regardless of whether `ogimage` exists. In fact (contrary to my PR name which I now realise is a misnomer), the only difference between my code and yours is that it's DRYer.

Reply:
>I can change this, but I'd like to point out that the behaviour resulting from my current changes is equivalent to the existing behaviour:

Thanks, Matthew for the observation. Please go ahead with dropping the existing behaviour. 
I wish to ensure that the meta tags are loaded only in the case any of the 2 things are defined.
This could prevent errors from being shown.

Reply:
Done